tests/test_kitsune.py

```diff
@@ -211,6 +216,18 @@ def test_fetch_empty(self):
 
         self.assertEqual(len(questions), 0)
 
+    @httpretty.activate
```

Comment:
I don't understand this test. I know that it is going to fail when an `HTTPError` exception is raised unless it is a 500 error, that would continue without failing. Your tests should take into account that behaviour.